### PR TITLE
Add logic to expose more retry info

### PR DESCRIPTION
**Pull request by MorrisLaw**
This is ~a WIP~ a PR for exposing more data from the retries.

### CODE REVIEW

**Comment by MorrisLaw** [comment]
Just updated the PR and decided to add a separate endpoint instead: `localhost:8080/retries`.

Using the `main.go` file I pushed up for testing, you'll get the following output at `localhost:8080/retries`:
https://example-retries-output.sfo2.digitaloceanspaces.com/Screen%20Shot%202019-12-13%20at%209.24.27%20AM.png

File: manager.go [hunk 1]
```diff
@@ -194,11 +197,83 @@ func (m *Manager) GetStats() (Stats, error) {
...
+// GetRetries returns the set of retry jobs for the manager
+func (m *Manager) GetRetries() (Retries, error) {
+	retries := Retries{}
+
+	storeRetries, err := m.opts.store.GetAllRetries()
+	if err != nil {
+		return retries, err
+	}
+
+	retryStats, err := m.opts.client.ZRange(m.opts.Namespace+storage.RetryKey, 0, 1).Result()
```
Comment (stefannegrea) [inline manager.go]: This number needs to be retrieved via `store.`

File: retries.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+type RetryJobStats struct {
+	Class        string `json:"class"`
+	ErrorMessage string `json:"error_message"`
+	FailedAt     string `json:"failed_at"`
+	JobID        string `json:"jid"`
+	Queue        string `json:"queue"`
+	RetryCount   int64  `json:"retry_count"`
+}
+
+func StartRetriesServer(port int) {
```
Comment (stefannegrea) [inline retries.go]: This probably needs to be registered on the same server as the stats manager.

File: storage/redis.go [hunk 3]
```diff
@@ -6,6 +6,7 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/bitly/go-simplejson"
```
Comment (stefannegrea) [inline storage/redis.go]: Could the build-in json library be used here?

File: manager.go (hunk 1 above)
Comment (MorrisLaw) [reply manager.go]: No longer using `ZRange`

File: storage/redis.go (hunk 3 above)
Comment (stefannegrea) [reply storage/redis.go]: Not a new dependency, resolving the comment.

**Pull request merged**